lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -22,16 +22,21 @@ def self.specify_options(c)
       end
 
       # Will modify post DOM tree
-      def self.download_images(title, post_hpricot, assets_folder)
+      def self.download_images(title, post_hpricot, assets_folder, published_at)
         images = (post_hpricot / "img")
         return if images.empty?
 
         Jekyll.logger.info "Downloading images for ", title
         images.each do |i|
           uri = i["src"]
 
-          i["src"] = format("{{ site.baseurl }}/%s/%s", assets_folder, File.basename(uri))
-          dst = File.join(assets_folder, File.basename(uri))
+          # Put the images into a /yyyy/mm/ subfolder to reduce clashes
+          datefolder = published_at.strftime("/%Y/%m")
+          assets_withdate_folder = format("%s%s", assets_folder, datefolder)
```

Comment:
`datefolder` and `withdate` aren't exactly valid words, are they?
```suggestion
          date_folder = published_at.strftime("/%Y/%m")
          assets_with_date_folder = format("%s%s", assets_folder, date_folder)
```

Reply:
Do ensure that all occurrences are renamed. I highlighted just a couple of them.

Reply:
agree with this. It wasn't consistent with other variables